### PR TITLE
Req view summary

### PR DESCRIPTION
**Pull request by Nauna**
new view for displaying capabilities requirements by displaying capability.of.section.help

### CODE REVIEW

File: src/foam/u2/crunch/CapabilityStore.js [hunk 1]
```diff
@@ -224,7 +225,7 @@ foam.CLASS({
           categoryId));
     },
     function signingOfficerQuestion() {
-      return this.auth.check(this.__subContext__, "businessHasSigningOfficer");
+      return this.auth.check(this.ctrl.__subContext__, 'businessHasSigningOfficer');
```
Comment (jlhughes) [inline src/foam/u2/crunch/CapabilityStore.js:228]: Assuming this is temporary?

File: src/foam/u2/crunch/CrunchController.js [hunk 2]
```diff
@@ -37,7 +37,8 @@ foam.CLASS({
 
   messages: [
     { name: 'CANNOT_OPEN_GRANTED', message: 'This capability has already been granted to you.' },
-    { name: 'CANNOT_OPEN_PENDING', message: 'This capability is awaiting approval, updates are not permitted at this time.' }
+    { name: 'CANNOT_OPEN_PENDING', message: 'This capability is awaiting approval, updates are not permitted at this time.' },
+    { name: 'CANNOT_OPEN_ACTION_PENDING', message: 'This capability is awaiting review by another user, updates are not permitted at this time.' }
```
Comment (jlhughes) [inline src/foam/u2/crunch/CrunchController.js]:
```suggestion
    { name: 'CANNOT_OPEN_ACTION_PENDING', message: 'This capability is awaiting review, updates are not permitted at this time.' }
```

File: src/foam/u2/crunch/CrunchController.js [hunk 3]
```diff
@@ -111,13 +112,13 @@ foam.CLASS({
...
-                .filter((wizardSection) =>
-                  wizardSection.ucj === null ||
-                  (
-                    ! wizardSection.ucj.status === this.CapabilityJunctionStatus.GRANTED &&
-                    ! wizardSection.ucj.status === this.CapabilityJunctionStatus.PENDING
-                  )
+              .filter((wizardSection) =>
+                wizardSection.ucj === null ||
+                (
+                  ! foam.util.equals(wizardSection.ucj.status, this.CapabilityJunctionStatus.GRANTED ) &&
```
Comment (kgrgreer) [inline src/foam/u2/crunch/CrunchController.js]: Enums can be compared with ==.

Comment (Nauna) [reply src/foam/u2/crunch/CrunchController.js]: So not suppose to have made this change - sorry

File: src/foam/u2/crunch/CrunchController.js [hunk 4]
```diff
@@ -169,15 +176,17 @@ foam.CLASS({
           this.EQ(this.UserCapabilityJunction.TARGET_ID, capabilityId)
         ));
       if ( ucj ) {
-        var statusGranted = ucj.status === this.CapabilityJunctionStatus.GRANTED;
+        var statusGranted = foam.util.equals(ucj.status, this.CapabilityJunctionStatus.GRANTED);
```
Comment (Nauna) [inline src/foam/u2/crunch/CrunchController.js]: oops

File: src/foam/u2/crunch/CapabilityStore.js (hunk 1 above)
Comment (Nauna) [reply src/foam/u2/crunch/CapabilityStore.js:228]: YES for sure, this will be a preRequisite of BusinessRegistration

**Review by jlhughes — APPROVED** [review]

**Pull request merged**